consensus/abci.go

```diff
@@ -123,17 +122,50 @@ func (abci *TendermintABCI) CheckTx(txBytes []byte) tmtAbciTypes.ResponseCheckTx
 	
 	tx, err := decodeRLP(txBytes)
 	if err != nil {
-		abci.logger.Error("Received invalid transaction", "tx", tx.Hash().String())
-		abci.metrics.CheckErrTxsTotal.Add(1, fmt.Sprint(abciTypes.ErrEncodingError.Code))
-		return tmtAbciTypes.ResponseCheckTx{Code: uint32(abciTypes.ErrEncodingError.Code), Log: err.Error()}
+		abci.logger.Error("Unable to decode RLP TX", "err", err.Error())
+		abci.metrics.CheckErrTxsTotal.Add(1, "INVALID_TX")
```

Comment:
[MINOR] Please encapsulate every error msg code into its own `error.go` file so we can identify where they have been used.

Reply:
u can see where they have been used, only here so no constant needed for now it just makes it harder to maintain, jumping through codebase.